js/dist/wp-seo-admin-gsc-320.js

```diff
@@ -15,6 +19,68 @@ jQuery( function() {
 			return window.open(auth_url, 'wpseogscauthcode', 'toolbar=no, location=no, directories=no, status=no, menubar=no, scrollbars=yes, resizable=no, copyhistory=no, width=' + w + ', height=' + h + ', top=' + top + ', left=' + left);
 		}
 	);
+
+	// Accessibility improvements for the Create Redirect modal dialog.
+	jQuery( '.wpseo-open-gsc-redirect-modal' ).click(
+		function( event ) {
```

Comment:
I think it would be better to rename `event` to `evt`, because `event` might be a reserved keyword in JavasScript.